airflow/ui/src/layouts/DagsLayout.tsx

```diff
@@ -16,5 +16,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { Box } from "@chakra-ui/react";
+import type { PropsWithChildren } from "react";
 
-export * from "./Runs";
+import { NavTabs } from "./Details/NavTabs";
+
+const tabs = [
+  { label: "Dags", value: "/dags" },
+  { label: "Runs", value: "/runs" },
+  { label: "Instances", value: "/instances" },
```

Comment:
The label could be "Task Instances" and value "task_instances" like other pages for consistency.